sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalog.scala

```diff
@@ -38,12 +38,12 @@ class JDBCTableCatalog extends TableCatalog with SupportsNamespaces with Logging
   private var dialect: JdbcDialect = _
 
   override def name(): String = {
-    require(catalogName != null, "The JDBC table catalog is not initialed")
+    require(catalogName != null, "The JDBC table catalog is not initialized")
```

Comment:
That's fine but mind taking a look at least this package and see if there are some more typos while we're here?